src/com/dre/brewery/recipe/CustomMatchAnyItem.java

```diff
@@ -228,4 +228,9 @@ public String toString() {
 			", loresize: " + (lore != null ? lore.size() : 0) +
 			'}';
 	}
+
+	@Override
+	public String displayName() {
+		return String.format("One of %s", String.join(", ", this.names));
```

Comment:
This could potentially be a big list of items, we should pick one or two at random here